projects/2023-NH/analysis.script_hadacek.R

```diff
@@ -1,19 +1,18 @@
-#######################
-# 2/27/23
+###
+# 3/27/23
 # Nick Hadacek
 # PSY4994V Final Project
-#######################
+###
 
-### LOADING PACKAGES ###
+# LOADING PACKAGES ####
 
 install.packages("ggplot2") #for data visualization
 install.packages("dplyr") #for data management
 install.packages("cowplot") #combining graphs if needed
 install.packages("psych") #easier correlation
 install.packages("apaTables") #APA-ready tables
-install.packages("haven") #i don't remember what this was for
```

Comment:
{haven} is typically used for reading in .sav file, but you have {rio} for this already.